cfg_mgmt/btp_service_binding.py

```diff
@@ -98,9 +118,21 @@ def get_service_bindings(self) -> list[ServiceBindingInfo]:
             id = item['id']
             name = item['name']
             instance_id = item['service_instance_id']
-            bindings.append(ServiceBindingInfo(id=id, name=name, service_instance_id=instance_id))
+            created_at = item['created_at']
+            updated_at = item['updated_at']
+            bindings.append(ServiceBindingInfo(
+                id=id, name=name, service_instance_id=instance_id, created_at=created_at, updated_at=updated_at))
         return bindings
 
+    def __response_json_to_sb_detailed_info(self, resp_json) -> ServiceBindingDetailedInfo:
```

Comment:
codestyle: "private" methods/functions should be marked w/ a single underscore. double-underscores are typically "reserved" for python-internals or -contracts 

consider using `dacite.from_dict` for deserialising `dict` -> `dataclass`

btw, using `dacite.config.Config`'s typehooks, you might even automatically deserialise timestamp-strings into `datetime.datetime` objects (if this is helpful/desirable), like so:


```
dacite.from_dict(
  data=resp_json,
  data_class=ServiceBindingDetailedInfo,
  config=dacite.config.Config(
    type_hooks={
     datetime.datetime: dateutil.parser.isoparse, # prefer dateutil.parser over python-default as the latter is buggy
   }
)
```

(for this to work, you would have to change the typehints for updated_at, created_at)